test/cri-containerd/createcontainer_test.go

```diff
@@ -792,3 +826,80 @@ func Test_CreateContainer_CPUShares_LCOW(t *testing.T) {
 	}
 	runCreateContainerTest(t, lcowRuntimeHandler, request)
 }
+
+func Test_CreateContainer_RunAs_LCOW(t *testing.T) {
+	pullRequiredLcowImages(t, []string{imageLcowK8sPause, imageLcowCosmos})
+
+	//sandbox reques
+	sandboxRequest := &runtime.RunPodSandboxRequest{
+		Config: &runtime.PodSandboxConfig{
+			Metadata: &runtime.PodSandboxMetadata{
+				Name:      t.Name() + "-Sandbox2",
+				Uid:       "0",
+				Namespace: testNamespace,
+			},
+			Linux: &runtime.LinuxPodSandboxConfig{
+				SecurityContext: &runtime.LinuxSandboxSecurityContext{
+					Privileged: true,
+				},
+			},
+		},
+		RuntimeHandler: lcowRuntimeHandler,
+	}
+
+	//container request
+	request := &runtime.CreateContainerRequest{
+		Config: &runtime.ContainerConfig{
+			Metadata: &runtime.ContainerMetadata{
+				Name: t.Name() + "-Container2",
+			},
+			Image: &runtime.ImageSpec{
+				Image: imageLcowCosmos,
+			},
+			// Hold this command open until killed
+			Command: []string{
```

Comment:
Sleep will eventually exit. I would prefer we use "cmd", "/c", "pause" and pass a `Stdin==true` or use a `ping` pattern like the others.

Reply:
This is an lcow container. I was gonna use "top", but looks like top doesn't work properly with the image (cosmosspark) I am using.

Reply:
Oh. Then can we use "tty==true" and "sh" as the command. That will also set the command to wait until closed